.github/workflows/ansible-test.yml

```diff
@@ -53,6 +71,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        ansible:
+          # It's important that Sanity is tested against all stable-X.Y branches
+          # Testing against `devel` may fail as new tests are added.
+          - stable-2.9
+          - stable-2.10
+          - devel
+        python:
+          - 2.7
+          - 3.7
+          - 3.8
+        exclude:
+          - python: 3.8  # blocked by ansible/ansible#70155
```

Comment:
That's not needed anywhere. 70155 only blocks `ansible-test sanity` when run without `--docker` in Python 3.8.